src/main/resources/scripts/rwa-2829_mi_data_fixes.sql

```diff
@@ -0,0 +1,216 @@
+  and created  < to_timestamp('2023-09-18 16:00:00','YYYY-MM-DD HH24:MI:SS') and state = 'COMPLETED';
+-- 4
+
+ select  count(*), update_action, termination_reason from cft_task_db.reportable_task rt where (first_assigned_date is null or first_assigned_date_time is null or wait_time_days is null or wait_time is null or
+  completed_date is null or completed_date_time is null or processing_time_days is null or processing_time is null or  handling_time_days is null or handling_time is null or
+  due_date_to_completed_diff_days is null or due_date_to_completed_diff_time is null or is_within_sla is null or
+  role_category_label is null or jurisdiction_label is null or case_type_label is null or state_label is null or final_state_label is null)
+  and created  < to_timestamp('2023-09-18 16:00:00','YYYY-MM-DD HH24:MI:SS') and state = 'COMPLETED' group by update_action, termination_reason;
+ -- 2	AddWarning	 [NULL]
+ -- 2	Complete	   completed
+
+ select count(*) from task_assignments ta where task_id in
+   (select task_id from tasks where created < to_timestamp('2023-09-18 16:00:00','YYYY-MM-DD HH24:MI:SS') and jurisdiction = 'PRIVATELAW'
+     and task_id in (select distinct task_id from task_history th  where created < to_timestamp('2023-09-18 16:00:00','YYYY-MM-DD HH24:MI:SS')))
+ -- 374
+
+ select count(*) from task_assignments ta where task_id in
+    (select task_id from tasks where created < to_timestamp('2023-09-18 16:00:00','YYYY-MM-DD HH24:MI:SS') and jurisdiction = 'CIVIL'
+      and task_id in (select distinct task_id from task_history th  where created < to_timestamp('2023-09-18 16:00:00','YYYY-MM-DD HH24:MI:SS')))
+  -- 5,674
+
+ select count(*) from task_assignments ta where task_id in
+    (select task_id from tasks where created < to_timestamp('2023-09-18 16:00:00','YYYY-MM-DD HH24:MI:SS') and jurisdiction = 'IA'
+      and task_id in (select distinct task_id from task_history th  where created < to_timestamp('2023-09-18 16:00:00','YYYY-MM-DD HH24:MI:SS')))
+  -- 74,697
+
+-- DATA CHANGE IMPLEMENTATION BY JURISDICTION - PRIVATELAW
+---------------------------------------------------------------
+CALL cft_task_db.mark_report_tasks_for_refresh(null,null,'PRIVATELAW',null,null, '2023-09-18 16:00:00'::timestamp);
+
+select count(*), state from tasks  where jurisdiction = 'PRIVATELAW' and report_refresh_request_time is not null and created < to_timestamp('2023-09-18 16:00:00','YYYY-MM-DD HH24:MI:SS') group by state
+--    Results should be like below
+--3	      PRIVATELAW	ASSIGNED
+--803  	  PRIVATELAW	TERMINATED
+--32  	  PRIVATELAW	UNASSIGNED
+
+select count(*), state from reportable_task rt where jurisdiction = 'PRIVATELAW' and report_refresh_time is not null and task_id in (select task_id from tasks where created < to_timestamp('2023-09-18 16:00:00','YYYY-MM-DD HH24:MI:SS')) group by state
+--    Results should be like below
+-- 3	    PRIVATELAW	ASSIGNED
+-- 185	  PRIVATELAW	TERMINATED
+-- 31   	PRIVATELAW	UNASSIGNED
```

Comment:
Probably need a query to find the ones that have failed which is a similar one to the one above on line 11.  You have the task_ids that need to be changed so we can find the ones that are in this table but have a null report_refresh_time.